tests/app/main/helpers/test_services.py

```diff
@@ -0,0 +1,104 @@
+import mock
+import pytest
+
+from app.main.helpers.services import copy_service_from_previous_framework
+
+
+class CustomAbortException(Exception):
+    """
+    Custom exception used in tests to mimick `aborts` behaviour of halting code execution. Catching this specific
+    exception in the tests allows us to know an exception is being thrown for the expected reason.
+    """
+    pass
+
+
+class TestCopyServiceFromPreviousFramework():
+    def setup_method(self):
+        self.abort_patch = mock.patch('app.main.helpers.services.abort')
+        self.abort = self.abort_patch.start()
+        self.abort.side_effect = CustomAbortException()
+
+        self.get_supplier_framework_info_patch = mock.patch('app.main.helpers.services.get_supplier_framework_info')
+        self.get_supplier_framework_info = self.get_supplier_framework_info_patch.start()
+        self.get_supplier_framework_info.return_value = True
```

Comment:
Returns True?

Reply:
Yeah, I'm just mocking out the guard clause. It just checks the truthyness off `get_supplier_framework_info`, doesn't actually do anything with the result. Do you think I should be returning an actual supplier framework?

Reply:
I think we should keep it as realistic as possible, yeah. If we don't have an api stub for a supplier framework and there's no easy thing to use from the supplier-fe test fixtures already, feel free to add a stub for it. ;)